Yoast/wpseo-woocommerce
# Fix OpenGraph deprecations

## Context


* Don't deprecate the whole class as there are still things in use.

## Summary

This PR can be summarized in the following changelog entry:

*  Fixes the OpenGraph class deprecations.

## Relevant technical choices:

* Removed the tests for the deprecated methods.

## Test instructions

This PR can be tested by following these steps:

* Check if all the methods with `_deprecated_function` have the doc comments `@deprecated`.
* Check that the deprecated methods are not used anywhere anymore (except the protected helper one: `get_brand_term_name`).

Fixes #
